libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -176,7 +176,7 @@ class AC_PrecLand
     AP_Int8                     _retry_behave;      // Action to do when trying a landing retry
     AP_Float                    _sensor_min_alt;     // PrecLand minimum height required for detecting target
     AP_Float                    _sensor_max_alt;     // PrecLand maximum height the sensor can detect target
-    AP_Int16                    _options;            // Bitmask for extra options
+    AP_Enum<Option>             _options;            // Bitmask for extra options
```

Comment:
Bitmasks are not enums because they can have multiple options set.

Reply:
Oh yes. Got it. Actually, someone asked me to do it this way in SIM_PRECLAND options when I was working on it. I missed it that time too. I have changed it to `AP_Int8` here. Thanks.